src/main/kotlin/kotlinx/kover/KoverPlugin.kt

```diff
@@ -302,7 +306,7 @@ private class BinaryReportCleanupAction(
 
         if (!taskExtension.isDisabled
             && !koverExtension.isDisabled
-            && !koverExtension.disabledProjects.contains(projectName)
+            && !koverExtension.disabledProjectsPaths.contains(projectPath)
```

Comment:
I see that extension now has both `disabledProjectsPaths` and `disabledProjects` with the same documentation. Is it intended? If so, they should clearly be distinct by their documentation

Reply:
Documentation on `disabledProjectsPaths` is not required, because this field is not available to the user.
But I will leave a correct dev comment.